src/util.h

```diff
@@ -338,4 +313,20 @@ static SECP256K1_INLINE int secp256k1_ctz64_var(uint64_t x) {
 #endif
 }
 
+/* Read a uint32_t in big endian */
+SECP256K1_INLINE static uint32_t secp256k1_read_be32(const unsigned char* p) {
+    return (uint32_t)p[0] << 24 |
```

Comment:
can you add more parens around the cast

Reply:
I can do it in a follow up PR if you insist but the code is correct given C's operator precedence.

Reply:
like this:

```c
(((uint32_t)p[0]) << 24) |
(((uint32_t)p[1]) << 16) |
(((uint32_t)p[2]) << 8) |
((uint32_t)p[3])
```

is much easier to read IMO

Reply:
Yeah, I agree, it may be easier to review for some reviewers because they don't need to to spend time looking up the rules. But given that it has been merged (i.e., review has been done), I don't think it's worth spending a PR on this. Feel free to open a PR if you disagree.  

(Sorry, I hadn't seen your comment earlier. I was spending some time testing this locally before pushing the merge. Your comment arrived after I had created the merge commit locally, but I believe before I pushed it. Then I didn't check github again before pushing the merge.)